ee/clickhouse/models/group.py

```diff
@@ -17,7 +17,9 @@ def create_group(
     group_key: str,
     properties: Optional[Dict] = {},
     timestamp: Optional[datetime.datetime] = None,
-) -> Group:
+    *,
+    clickhouse_only: bool = False,
```

Comment:
instead let's create a function to create the group for CH and call that in here and wherever you need.

Reply:
Makes more sense, thanks